tests/migration/migration.go

```diff
@@ -1239,7 +1239,26 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 			}, 30*time.Second).Should(Not(BeNil()))
 			By("waiting for the dv import to pvc to finish")
 			libstorage.EventuallyDV(dv, 180, HaveSucceeded())
-			tests.ChangeImgFilePermissionsToNonQEMU(pvc)
+
+			By("changing disk.img permissions to non qemu")
+			args := []string{fmt.Sprintf(`chmod 640 %s && chown root:root %s && sync`, filepath.Join(libstorage.DefaultPvcMountPath, "disk.img"), filepath.Join(libstorage.DefaultPvcMountPath, "disk.img"))}
+			pod := libstorage.RenderPodWithPVC("change-permissions-disk-img-pod", []string{"/bin/bash", "-c"}, args, pvc)
+
+			// overwrite securityContext
+			rootUser := int64(0)
+			pod.Spec.Containers[0].SecurityContext = &k8sv1.SecurityContext{
+				Capabilities: &k8sv1.Capabilities{
+					Drop: []k8sv1.Capability{"ALL"},
+				},
+				Privileged:   pointer.Bool(true),
```

Comment:
Please use here the kubevirt pointer

Reply:
This is addressed in a subsequent commit but I'll make sure to clean up the order a bit.